packages/teams-js/src/public/dialog.ts

```diff
@@ -161,23 +167,29 @@ export namespace dialog {
       botUrlDialogInfo: BotUrlDialogInfo,
       submitHandler?: DialogSubmitHandler,
       messageFromChildHandler?: PostMessageChannel,
-    ): PostMessageChannel {
+    ): void {
       ensureInitialized(FrameContexts.content, FrameContexts.sidePanel, FrameContexts.meetingStage);
 
       if (messageFromChildHandler) {
         registerHandler('messageForParent', messageFromChildHandler);
       }
-
       sendMessageToParent('tasks.startTask', [botUrlDialogInfo], (err: string, result: string | object) => {
         submitHandler({ err, result });
         removeHandler('messageForParent');
       });
+    }
 
+    /**
+     *  Send message to the dialog from the parent
+     *
+     * @param message - The message to send
+     */
+    export function sendMessageToDialog(
```

Comment:
I think, we don't need a separate a function for `bot` since this not different than the other outside function. What do you think?

Reply:
I didn't know if this was needed or not, I will remove it. Good catch :)